src/main/java/BalkanBot.java

```diff
@@ -0,0 +1,202 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class BalkanBot {
+    private static final String line = "------------------------------------------";
+
+    public static void printComplexTask(Task[] arr, int current) {
+        System.out.println(line);
+        System.out.println("Got it I've now added this task:");
+        System.out.println(arr[current - 1].toString());
+        System.out.println("Now you have " + current + " task(s) in the list.");
+        System.out.println(line);
+    }
+
+    public static void printDeletion(Task deletedTask, int current) {
+        System.out.println(line);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(deletedTask.toString());
+        System.out.println("Now you have " + current + " task(s) in the list.");
+        System.out.println(line);
+    }
+
+    public static void main(String[] args) {
+        String line = "------------------------------------------";
+        Task[] listOfTasks = new Task[100];
```

Comment:
Rather than using a java array, perhaps it would be easier and more scalable to use a ArrayList instead?
```suggestion
        ArrayList<Task> listOfTasks = new ArrayList<>();
```